CONTRIBUTING-GUIDE.md

```diff
@@ -0,0 +1,32 @@
+
+## Keploy Ecosystem: A Gateway to Contributing
+
+| Web Presence                                  | Description                                                                                                                                                                                                                                                                                                                                              | GitHub Repository                                                       | Contribution Guide                                                                                 | Issue Tracker                                                         | Latest Release                                                       |
+| --------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| [Keploy.io](https://keploy.io/)               | Keploy, a developer-centric backend testing tool, offers features like combined test coverage, EBPF instrumentation, CI/CD integration, and multi-purpose mocks.                                                                                                                                                                                         | [keploy/keploy](https://github.com/keploy/keploy)                       | [Contribute Here](https://github.com/keploy/keploy/blob/main/CONTRIBUTING.md)                      | [View Issues](https://github.com/keploy/keploy/issues)                | [Release Info](https://github.com/keploy/keploy/releases)            |
```

Comment:
Change Web Presence to "Project Name"

Project Name: Keploy.io should be Keploy Server/Agent.

Issue Tracker - https://github.com/keploy/keploy/issues?q=is%3Aissue+is%3Aopen+label%3Akeploy Include the label of that repository so that people don't get confused.

Also, please don't add a link to something that doesn't have a hosted URL. 

@prajwalpd7 @Sonichigo this needs to changed respectively for every project.